Document Agenda application (#288)
exoplatform/exo-documentation: docs/Calendar.rst

==========================

You can change the Agenda preferences as follows:

1. Open the Agenda Settings form via either of the following ways:

   -  **The first way**: Click |AGENDA15| on the Agenda toolbar.

   -  **The second way**: Click |AGENDA16|, then select edit Agenda Settings button.

In this new setting, you are able to edit your Agenda preferences by configuring: 

  - Agenda default view 

  - Week Starting day 

  - Working time 

  - Default Reminders

By clicking on the edit button, a drawer is opened to edit these preferences. 

The Agenda Settings form appears:

|AGENDA25|

2. Change your desired preferences.

**i.** In the Settings tab, you can change the Agenda general
       settings:

.. _ConnectPersonalCalendar:

==========================
Connect Personal Calendar
==========================

You are now able to connect your personal calendar and display your personal events in your **Schedule meeting** view to help you scheduling your meeting or event.

Where Can I Connect to my Personal Calendar ? 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You can Connect to your personal Calendar via either of these 3 ways:

**The first way**

> You are invited to open your Settings page and just below the Agenda section, you  find a new section Connect your personal Calendar.

Reviewer: ```suggestion
Go to your personal your Settings and just below the Agenda section, find  section : Connect your personal Calendar.
```

**The second way**

A new section is added to the Choose a date stepper in which you find if you are connected to your Google personal Calendar or not.

You need to click on the *Connect your personal calendar* button to choose it.

|AGENDA27|

**The third way**

A new section is added to any event view you are invited to participate to in which you find if you are connected to your Google personal Calendar or not.

You need to click on the *Connect your personal calendar* button to choose it.

|AGENDA28|

.. note:: If no personal calendar connector is displayed in the Personal Calendar drawer, an information message is displayed. 
   |AGENDA29|.
   you can take a look on :ref:`Manage Personal Calendar <ManagePersonalCalendar>` for 
   more detail about enabling and displaying Personal Calendars.
  

How can I connect my personal calendar ?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You can simply click on the *edit icon* in the Settings page, or Connect to my personal Calendar in the *Choose a date* stepper or *Event view*.

A drawer that contains the Calendars connectors is displayed.

You have 2 choices: Google Calendar and Microsoft Outlook. 

|AGENDA30|

> You are invited to click on connect in front of one of these choices. 

Reviewer: ```suggestion
Click on Connect in front of one of these options. 
```

If you are not connected to your personal calendar, you can click on You are invited to connect to your personal Calendar and the Connect to personal calendar drawer is opened so that you can connect to it.

When you click on connect, a popup of your connected email in the browser is displayed, or you can connect to another email.

|AGENDA31|

.. note:: You are allowed to connect to only one personal calendar. 

Once connected, you will have your personal connected Google email and Google avatar for Google account, and same for Outlook account.

|AGENDA32|

How can I view my personal events ?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You can view your personal events via either of these 2 views:

**Choose a date stepper**:

> As long as your personal calendar is connected to the digital workplace, you are able to view your personal events in the Choose a date step when you create a new event so that you are aware about your events in the chosen space to create the event into and your Google personal events.

Reviewer: ```suggestion
As long as your personal calendar is connected to the digital workplace, you can view your personal events in the Choose a date step when you create a new event so that you are aware about your events in the chosen space to create the event into and your Google personal events.
```
